### PR TITLE
Config for displaying events with same template

### PR DESCRIPTION
**Pull request by llmlks**

### CODE REVIEW

**Review by niklasmaki — CHANGES_REQUESTED** [review]
I think the code changes are fine and the functionality works as it should 👍 

However, I get an error if I go to the map view with either the Basel dataset or the SCEDC dataset. Something about a missing template id.

Also another thing. If I understand correctly, when you select a template id, all the events with that template id from the entire dataset are shown. The date range the user has selected is completely ignored. This is not necessarily a problem, but it's kind of confusing. There is no way the user knows what data is shown. At least I initially thought that it only shows the event from the selected date range.

What do you think?

**Comment by llmlks** [comment]
I'll look into the errors. 

I thought about the problem of what data to show, and ended up thinking that seeing only data from selected time range would be very restrictive. Though, I was actually only considering the time frame combined with the time slider and failed to take into account using only the date range. Currently, if the user selects a template ID from the configurations and applies the changes, the time configurations are ignored, and the time slider is hidden from under the map. Maybe it could be set optional (show only events in the selected time frame/ across whole dataset)? Additionally, adding some conditional stylings on the time configurations would help clarify the issue (e.g. fade them a little, or otherwise visualise they are disabled)?

**Comment by niklasmaki** [comment]
Yeah, I think even a slight visual hint would be helpful for the user.
Adding more configurations might also be confusing.

File: components/config/map_config.py [hunk 1]
```diff
@@ -1,28 +1,78 @@
...
 
     Keyword arguments:
     min_date -- The minimum date allowed to pick with the datepicker.
     max_date -- The maximum date allowed to pick with the datepicker.
     default_end_date -- The default end date for the datepicker. The
         default start date is the `min_date`.
+    templates -- A list of template IDs to select from.
     columns -- The available columns in the uploaded data
+    templates -- A list of template IDs to select from.
     """
```
Comment (niklasmaki) [inline components/config/map_config.py:30]: The templates docstring is twice here.

**Review by niklasmaki — CHANGES_REQUESTED** [review]
I think that using the selected time range is a good solution. And I like disabling the timestep selector. Now it's more clear which configurations can be used with a template id.

However, the color and size dropdowns are now broken. For some reason they contain template ids as well.

Also, now I'm getting an error with the SCEDC and Otaniemi datasets:
```
Traceback (most recent call last):
  File "C:\Users\Niklas\Documents\Data Science Project I\QuakeWatch\index.py", line 44, in display_page
    return quake_map.get_layout(session_id)
  File "C:\Users\Niklas\Documents\Data Science Project I\QuakeWatch\views\quake_map.py", line 49, in get_layout
    dbc.Col(map_config.get_component(
  File "C:\Users\Niklas\Documents\Data Science Project I\QuakeWatch\components\config\map_config.py", line 32, in get_component
    size_picker.get_component(columns),
  File "C:\Users\Niklas\Documents\Data Science Project I\QuakeWatch\components\config\size_picker.py", line 21, in get_component
    options=[
TypeError: 'NoneType' object is not iterable
```

**Review by niklasmaki — APPROVED** [review]
Now everything works 👍

**Pull request merged**